### ISSUE TITLE
[BUG] `Error 'pywin32_system32`  in win10 enterprise ltsc

### ISSUE THREAD

**Issue by mozhe-fhy** [dashingsoft/pyarmor#1239]
```
(py310) PS D:\python> pyarmor gen --mix-str --assert-call --assert-import --restrict --pack .\dist\run.exe run.py
INFO     Python 3.10.9
INFO     Pyarmor 8.2.0 (pro), 005126, Mozhe
INFO     Platform windows.x86_64
INFO     implicitly set output to ".pyarmor\pack\dist"
INFO     extracting bundle ".\dist\run.exe"
.......
INFO     replace item "xmlrpc.client"
INFO     replace item "zipfile"
INFO     replace item "zipimport"
ERROR    'pywin32_system32
```
python 切换到3.11.3也是一样的问题

附件为error 
[pyarmor.error.log](https://github.com/dashingsoft/pyarmor/files/11461768/pyarmor.error.log)
log

**Comment by jondy** [comment]
Got it, I'll check it.

**Comment by kr1tx** [comment]
Same, Got it after upgrading pyinstaller to 5.10.1
```
INFO     Python 3.10.8
INFO     Pyarmor 8.2.0 (pro), 00xxxx, non-profits(TBD)
INFO     Platform windows.x86_64
INFO     implicitly set output to ".pyarmor\pack\dist"
INFO     extracting bundle "dist\Script.exe"
INFO     search inputs ...
INFO     find script Script.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
INFO     write .pyarmor\pack\dist\pyarmor_runtime_00xxxx\pyarmor_runtime.pyd
INFO     patch runtime file
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "Script"
INFO     obfuscating file Script.py
INFO     start to generate bcc code ...
INFO     generate bcc code successfully
INFO     write .pyarmor\pack\dist\Script.py
INFO     obfuscate scripts OK
INFO     repacking bundle "dist\Script.exe"
INFO     obfuscated scripts at ".pyarmor\pack\dist"
INFO     entry script name is "Script.py"
INFO     repacking "PYZ-00.pyz"
INFO     replace item "pyarmor_runtime_00xxxx"
ERROR    'pywin32_system32'
```

**Comment by jondy** [comment]
Fixed in 8.2.1
